gnosis/safe/addresses.py

```diff
@@ -542,6 +542,24 @@
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 17, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 18, "1.3.0"),
     ],
+    EthereumNetwork.FANTOM_OPERA: [
+        ("0x41675C099F32341bf84BFc5382aF534df5C7461a", 65771679, "1.4.1+L2"),
```

Comment:
We are working on support 1.4.1 contracts on safe-eth-py.

Reply:
So should I remove 1.4.1 from this PR to make it thru?

Reply:
yes at least for now.

Reply:
Thanks for your feedback. Removed 1.4.1 version of contracts.